File: src/main/java/com/temelyan/pomoapp/repository/PomoRepository.java
package com.temelyan.pomoapp.repository;

import com.temelyan.pomoapp.model.Pomo;

import java.time.LocalDate;
import java.util.List;

public interface PomoRepository {
    Pomo save(Pomo pomo, int taskId);

    List<Pomo> getAllForUserInDateRange(LocalDate fromDate, LocalDate toDate, int userId);
}

File: src/main/java/com/temelyan/pomoapp/repository/PomoRepository.kt
package com.temelyan.pomoapp.repository

import com.temelyan.pomoapp.model.Pomo

import java.time.LocalDate

interface PomoRepository {
    fun save(pomo: Pomo, taskId: Int): Pomo

    fun getAllForUserInDateRange(fromDate: LocalDate, toDate: LocalDate, userId: Int): List<Pomo>
}
